src/routes/Project/components/Table.js

```diff
@@ -1,11 +1,16 @@
-import React from 'react'
+import React from 'react';
 
 export const Table = (props) => (
```

Comment:
where do these props come from? I see you instantiating Table in the file above as `<Table/>,` don't you need to give it props there or do i misunderstand.